types/chrome/index.d.ts

```diff
@@ -4204,7 +4204,30 @@ declare namespace chrome.input.ime {
      * If you specify the callback parameter, it should be a function that looks like this:
      * function(boolean success) {...};
      */
-    export function setAssistiveWindowProperties(parameters: object, callback?: (success: boolean) => void): void;
+    export function setAssistiveWindowProperties(parameters: {
+            contextID: number,
+            properties: chrome.input.ime.AssistiveWindowProperties
+        }, callback?: (success: boolean) => void): void;
+    /**
+     * Highlights/Unhighlights a button in an assistive window.
+     * @param {{
+     *   contextID: number,
+     *   buttonID: chrome.input.ime.AssistiveWindowButton,
+     *   windowType: chrome.input.ime.AssistiveWindowType,
+     *   announceString?: string,
+     *   highlighted: boolean
+     * }} parameters
```

Comment:
I think it this `@param` comment is not necessary because it is explained enough by the type definition.
If you want to write `@param`, how about do that like this:
```
/**
 * Highlights/Unhighlights a button in an assistive window.
 * @param parameters
 * @param callback Called when the ...
```

Reply:
Thanks pokutuna, comments removed :D